src/index.css

```diff
@@ -369,3 +369,13 @@ div.monokleEditorErrorRefGlyphClass.monokleEditorUnsatisfiedRefGlyphClass::after
 ::-webkit-scrollbar-thumb:hover {
   background: #242424;
 }
+
+.filters-preset-dropdown .ant-select-item-option-content {
```

Comment:
couldn't this be done in a styled component, or is it because the dropdown is rendered outside the component tree?

Reply:
Exactly, couldn't be done with a styled component